docs/examples/vector_stores/pinecone_auto_retriever.ipynb

```diff
@@ -48,7 +48,7 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.        default_empty_query_vector=[0] * 1536
</pre><p>ic what is point for empty query vector now</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_index/pull/9647/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>